src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card  {
+  margin: 0 auto;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)